### PR TITLE
Remove unnecessary IE warning

### PR DESCRIPTION
**Pull request by dacook**
I was waiting for something to load and noticed we have a warning for IE.
It brings me great pleasure to remove this. I've been waiting my whole career: IE is finally gone!!

Also, sed is your friend. `gsed /ie_warning/d -i config/locales/*.yml`

#### What? Why?

Removes unnecessary code and reduces network traffic.

#### What should we test?
No need to manual test I think.

#### Release notes

Changelog Category: Technical changes

#### Dependencies
IE was retired on 15th June 2022
https://blogs.windows.com/windowsexperience/2022/06/15/internet-explorer-11-has-retired-and-is-officially-out-of-support-what-you-need-to-know/

### CODE REVIEW

**Review by jibees — APPROVED** [review]
When I was young, I had to add some code to make IE compatible ; now that I'm old we're removing that IE-specific code. Finally, that time is coming.

I was just curious if we need to delete all the keys in all i18n files, or simply removing those keys in `en.yml` is enough since it's synchronized via Transifex.

File: app/views/layouts/darkswarm.html.haml [hunk 1]
```diff
@@ -36,10 +36,6 @@
     = action_cable_meta_tag
 
   %body{ class: body_classes, "body-scroll": "true", "data-turbo": "false" }
-    / [if lte IE 8]
-      = render partial: "shared/ie_warning"
-      = javascript_include_tag "iehack"
```
Comment (mkllnk) [inline app/views/layouts/darkswarm.html.haml:41]: We can remove that JS file, too. It's referenced in config/application.rb and located in app/assets/javascripts/iehack.js.

File: config/locales/ar.yml [hunk 2]
```diff
@@ -1703,12 +1703,6 @@ ar:
   card_has_been_removed: "تمت إزالة بطاقتك (الرقم: %{number})"
   card_could_not_be_removed: عذرًا ، تعذرت إزالة البطاقة
   invalid_credit_card: "بطاقة الائتمان غير صالحة"
-  ie_warning_headline: "متصفحك غير محدث :-("
-  ie_warning_text: "للحصول على أفضل تجربة لشبكة الغذاء المفتوح ، نوصي بشدة بترقية متصفحك:"
-  ie_warning_chrome: تحميل متصفح كروم
-  ie_warning_firefox: تحميل متصفح فايرفوكس
-  ie_warning_ie: ترقية Internet Explorer
-  ie_warning_other: "لا يمكنك ترقية متصفحك؟ جرب شبكة الغذاء المفتوح على هاتفك الذكي :-)"
```
Comment (mkllnk) [inline config/locales/ar.yml:1711]: We should only change en.yml. The others will be updated by Transifex and we don't want merge conflicts.

**Review by mkllnk — CHANGES_REQUESTED** [review]
What a lovely day!

Finally!

File: config/locales/ar.yml (hunk 2 above)
Comment (dacook) [reply config/locales/ar.yml]: Thanks for confirming this, good to know.

**Comment by dacook** [comment]
Thanks for the feedback, I must have been so excited I missed the JS file. I also missed some image files.
The commit has now been updated (it was simpler than adding a new one).

**Review by mkllnk — APPROVED** [review]
:+1:

**Pull request merged**